portfolio/src/main/webapp/script.js

```diff
@@ -56,22 +65,92 @@ async function getServerContent() {
  */
 function createCommentElement(comment) {
   const divElement = document.createElement('div');
-  divElement.setAttribute("class", "comment-div");
+  divElement.setAttribute('class', 'comment-div');
   
-  var userP = document.createElement("span");
+  const userP = document.createElement("span");
   userP.innerText = comment.user; 
-  userP.setAttribute("class", "user-attr");
+  userP.setAttribute('class', 'user-attr');
 
-  var dateP = document.createElement("span");
+  const dateP = document.createElement('span');
   dateP.innerText = comment.commentDate; 
-  dateP.setAttribute("class", "date-attr");
+  dateP.setAttribute('class', 'date-attr');
+
+  const deleteButton = document.createElement('button');
+  deleteButton.innerHTML = '<i class="fas fa-trash-alt"></i>';
+  deleteButton.setAttribute('class', 'del-attr');
+  deleteButton.addEventListener('click', () => {
+    deleteComment(comment);
+  });
 
-  var textP = document.createElement("p"); 
+  const textP = document.createElement('p');
```

Comment:
what is 'P' meant to communicate? Parameter?

Reply:
The 'P' was meant to show that it was a paragraph element, but I can see that's unclear, so I'll change it to textParagraph.